src/search.css

```diff
@@ -114,6 +114,7 @@
   width: 100%;
   padding: 6px;
   line-height: 1;
+  color: black; /* Fix https://github.com/readthedocs/addons/issues/420 */
```

Comment:
This should use a variable instead of hard coding the value. I looked into this and the issue is that the computed value is from the element style, which has higher specificity than the inherited value from the parent div. So reiterating the inherited value here is probably best either way.

![image](https://github.com/user-attachments/assets/e06d2030-3470-406d-8a22-c1e6399a2a4d)


```suggestion
  color: var(--readthedocs-search-color, rgb(64, 64, 64));
```